contrib/side-cars/stacks-block-delay-event-trigger.py

```diff
@@ -0,0 +1,226 @@
+"""
```

Comment:
Can you give a motivating example as to what feature(s) this side-car enables?

Reply:
2 things come to mind here if the threshold is met as defined in the script:
1. this could trigger a command to run an external script to move some btc from a hot wallet to the miner address
2. adds the ability to send a notification in the case that no blocks have been mined over a specified timeframe

I don't expect a lot of miners would care to use this, but i can see the benefit it adds.

Reply:
Yes case 1 was the original motivation for this script.

Reply:
Since we expect this script will not be run by most nodes, is there somewhere better for me to put it?

Reply:
No, this is fine.  `contrib/` is a grab-bag of ancillary tooling that might be useful to some folks, but receives best-effort support from the tools' original authors.